### PR TITLE
chore: added warning for unsupported node.js --experimental-loader flag

### PR DESCRIPTION
**Pull request by aryamohanan**
The PR includes logic to detect the usage of the '--experimental-loader' flag with Node.js versions 18.19 and above, and then
 added the warning message to clarify that support for the experimental loader flag has been dropped due to breaking changes in Node.js versions 18.19 and 20.0. Also provided instructions to use the '--import' flag for Instana monitoring and included a reference to the Instana documentation for further guidance.

### CODE REVIEW

File: packages/core/src/tracing/index.js [hunk 1]
```diff
@@ -160,6 +165,14 @@ exports.preInit = function preInit(preliminaryConfig) {
  * @param {CollectorPIDStore} _processIdentityProvider
  */
 exports.init = function init(_config, downstreamConnection, _processIdentityProvider) {
+  if (hasExperimentalLoaderFlag()) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      "Instana no longer supports the '--experimental-loader' flag starting from Node.js 18.19.0. " +
+        `The current Node.js version is ${process.version}. To be monitored by Instana, please use ` +
+        "the '--import' flag instead. Refer to the Instana documentation for further details."
```
Comment (kirrg001) [inline packages/core/src/tracing/index.js:173]: issue: Please add docs link

File: packages/core/src/tracing/index.js [hunk 2]
```diff
@@ -160,6 +165,14 @@ exports.preInit = function preInit(preliminaryConfig) {
  * @param {CollectorPIDStore} _processIdentityProvider
  */
 exports.init = function init(_config, downstreamConnection, _processIdentityProvider) {
+  if (hasExperimentalLoaderFlag()) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      "Instana no longer supports the '--experimental-loader' flag starting from Node.js 18.19.0. " +
```
Comment (kirrg001) [inline packages/core/src/tracing/index.js:171]: suggestion: I'd suggest it differently. e.g. "Node.js invented breaking changes in 18.19 & 20....thats why we had to drop the support for experiemntal loader as well.."

**Review by kirrg001 — CHANGES_REQUESTED** [review]

File: packages/core/src/tracing/index.js (hunk 2 above)
Comment (aryamohanan) [reply packages/core/src/tracing/index.js]: Upfated

File: packages/core/src/tracing/index.js (hunk 1 above)
Comment (aryamohanan) [reply packages/core/src/tracing/index.js]: added

File: packages/core/src/tracing/index.js [hunk 3]
```diff
@@ -160,6 +164,16 @@ exports.preInit = function preInit(preliminaryConfig) {
  * @param {CollectorPIDStore} _processIdentityProvider
  */
 exports.init = function init(_config, downstreamConnection, _processIdentityProvider) {
+  if (hasExperimentalLoaderFlag()) {
```
Comment (kirrg001) [inline packages/core/src/tracing/index.js:167]: suggestion: add a hint for v4 release e.g. // consider removing in v4

File: packages/core/src/tracing/index.js [hunk 4]
```diff
@@ -160,6 +164,16 @@ exports.preInit = function preInit(preliminaryConfig) {
...
+    // eslint-disable-next-line no-console
+    console.warn(
+      'Node.js introduced breaking changes in versions 18.19.0 and above, leading to the discontinuation of support ' +
+        `for the experimental loader flag by Instana. The current Node.js version is ${process.version}. ` +
```
Comment (kirrg001) [inline packages/core/src/tracing/index.js:171]: suggestion: for the experimental loader flag `--experimental-loader` flag

File: packages/core/src/tracing/index.js [hunk 5]
```diff
@@ -160,6 +164,16 @@ exports.preInit = function preInit(preliminaryConfig) {
...
+        "To ensure monitoring by Instana, please use the '--import' flag instead. For more information, " +
```
Comment (kirrg001) [inline packages/core/src/tracing/index.js:172]: suggestion: use the wording "tracing" such as "Please switch to the --import flag to ensure that tracing works again". (just an example)

**Review by kirrg001 — APPROVED** [review]
LGTM. Just left some final suggestions :)

**Pull request merged**